### PR TITLE
Update Models.json

### PR DESCRIPTION
**Pull request by diazandr3s**
Update all labels.csv files in all models and upload 3 new models trained on the TotalSegmentatorV2 dataset following this grouping: https://github.com/wasserth/TotalSegmentator/blob/9132e29ed991866aed3c1ede9f3d377d4c0c083f/totalsegmentator/map_to_binary.py#L320

### CODE REVIEW

File: MONAIAuto3DSeg/Resources/Models.json [hunk 1]
```diff
@@ -25,25 +27,57 @@
...
-        }
+        },
+        {
+            "title": "Cardiac",
+            "description": "class_map_part_cardiac: Trained on TotalSegmentatorV2 dataset.",
+            "subject": "human",
+            "imagingModality": "CT",
+            "versions":
+            [
+                { "url": "https://github.com/lassoan/SlicerMONAIAuto3DSeg/releases/download/Models/cardiac-V2.0.0.zip" }
```
Comment (lassoan) [inline MONAIAuto3DSeg/Resources/Models.json:59]: please use lowercase `v` for version separator for consistency

File: MONAIAuto3DSeg/Resources/Models.json [hunk 2]
```diff
@@ -25,25 +27,57 @@
...
+            ]
+        },   
+        {
+            "title": "Organs",
+            "description": "class_map_part_organs: Trained on TotalSegmentatorV2 dataset.",
+            "subject": "human",
+            "imagingModality": "CT",
+            "versions":
+            [
+                { "url": "https://github.com/lassoan/SlicerMONAIAuto3DSeg/releases/download/Models/organs-V2.0.0.zip" }
```
Comment (lassoan) [inline MONAIAuto3DSeg/Resources/Models.json:69]: please use lowercase `v` for version separator for consistency

File: MONAIAuto3DSeg/Resources/Models.json [hunk 3]
```diff
@@ -25,25 +27,57 @@
...
+            ]
+        },  
+        {
+            "title": "Vertebrae",
+            "description": "class_map_part_vertebrae: Trained on TotalSegmentatorV2 dataset.",
+            "subject": "human",
+            "imagingModality": "CT",
+            "versions":
+            [
+                { "url": "https://github.com/lassoan/SlicerMONAIAuto3DSeg/releases/download/Models/vertebrae-V2.0.0.zip" }
```
Comment (lassoan) [inline MONAIAuto3DSeg/Resources/Models.json:79]: please use lowercase `v` for version separator for consistency

File: MONAIAuto3DSeg/Resources/Models.json [hunk 4]
```diff
@@ -25,25 +27,57 @@
...
+            "description": "class_map_part_cardiac: Trained on TotalSegmentatorV2 dataset.",
+            "subject": "human",
+            "imagingModality": "CT",
+            "versions":
+            [
+                { "url": "https://github.com/lassoan/SlicerMONAIAuto3DSeg/releases/download/Models/cardiac-V2.0.0.zip" }
+            ]
+        },   
+        {
+            "title": "Organs",
```
Comment (lassoan) [inline MONAIAuto3DSeg/Resources/Models.json:63]:
```suggestion
            "title": "Abdominal organs TS2",
```

File: MONAIAuto3DSeg/Resources/Models.json (hunk 2 above)
Comment (lassoan) [inline MONAIAuto3DSeg/Resources/Models.json:69]:
```suggestion
                { "url": "https://github.com/lassoan/SlicerMONAIAuto3DSeg/releases/download/Models/abdominal-organs-v2.0.0.zip" }
```

**Review by lassoan — COMMENTED** [review]
Looks good, just added a few trivial comments.

File: MONAIAuto3DSeg/Resources/Models.json (hunk 1 above)
Comment (diazandr3s) [reply MONAIAuto3DSeg/Resources/Models.json]: Thanks! Just updated the name!

File: MONAIAuto3DSeg/Resources/Models.json (hunk 2 above)
Comment (diazandr3s) [reply MONAIAuto3DSeg/Resources/Models.json]: Thanks! Just updated the name!

**Pull request merged**